bin/ch/WScriptJsrt.cpp

```diff
@@ -725,7 +725,7 @@ bool WScriptJsrt::InstallObjectsOnObject(JsValueRef object, const char* name, Js
     memcpy(path, msg, (size_t)str_len);          \
     path[str_len] = char(0)
 
-void GetBinaryLocation(char *path, const uint32_t size)
+void GetBinaryLocation(char *path, const unsigned size)
```

Comment:
`unsigned int` is just more clear, but since it's equivalent, there's no actual problem or advantage/disadvantage from a functionality perspective. We can table that.

I'm more concerned about the size_t versus unsigned and the bitness of the type of this param and the value passed in on various systems.

Since the value passed in is a size_t what happens if that value is larger than 32 bits?